test/cmd/legacy-script.sh

```diff
@@ -67,14 +67,14 @@ CTLRMGR_PORT=${CTLRMGR_PORT:-10252}
 PROXY_HOST=127.0.0.1 # kubectl only serves on localhost.
 
 IMAGE_NGINX="k8s.gcr.io/nginx:1.7.9"
```

Comment:
is this one unused?

Reply:
No, actually it's used in current file.
And because of this it's okay to not export it.